src/Query/ResultPrinters/TableResultPrinter.php

```diff
@@ -118,7 +118,7 @@ protected function getResultText( QueryResult $res, $outputMode ) {
 		// building headers
 		if ( $this->mShowHeaders != SMW_HEADERS_HIDE ) {
 			$isPlain = $this->mShowHeaders == SMW_HEADERS_PLAIN;
-			foreach ( $res->getPrintRequests() as /* SMWPrintRequest */ $pr ) {
+			foreach ( $res->getPrintRequests() as /* \SMW\Query\PrintRequest */ $pr ) {
```

Comment:
This doc is superfluous since `getPrintRequests` documents its return type. You could remove it

Reply:
Thanks, done.